pyproject.toml

```diff
@@ -87,6 +87,7 @@ test = [
     "safety",
     "numpy",
     "scipy",
+    "pandas",
```

Comment:
ah, for the optional `pandas` output, I see.
As long as we don't make it a hard dep, it's fine, I suppose.

Reply:
Not a blocking comment.

Reply:
Yep, these aren't installed when the package is installed. Just in the CI and optionally by users, so this should let us handle other soft dependencies this way too (though we'd like to minimize them too).